### PR TITLE
Visitor can view a collection of products (menu)

### PR DESCRIPTION
**Pull request by tochman**
PT feature: https://www.pivotaltracker.com/story/show/181399296

Screenshot 2022-03-03 at 12 09 03

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import React from "react";
+import Menu from './components/Menu'
```
Comment (maxarvid) [inline src/App.js]: format code

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import './index.css'
```
Comment (maxarvid) [inline src/index.js]: format code

**Review by maxarvid — COMMENTED** [review]
Mixes single and double quotes, pls harmonize.

File: src/index.js (hunk 2 above)
Comment (tochman) [reply src/index.js]: fixed

File: src/App.js (hunk 1 above)
Comment (tochman) [reply src/App.js]: fixed

**Review by maxarvid — APPROVED** [review]
LGTM

**Pull request merged**